supplements/dnd-beyond/descent-into-the-lost-caverns-of-tsojcanth.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<update version="0.0.1">
+			<file name="descent-into-the-lost-caverns-of-tsojcanth.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/dnd-beyond/descent-into-the-lost-caverns-of-tsojcanth.xml" />
+		</update>
+	</info>
+
+	<element name="Descent into the Lost Caverns of Tsojcanth" type="Source" source="Core" id="ID_WOTC_DLCT_SOURCE_DESCENT_INTO_THE_LOST_CAVERNS_OF_TSOJCANTH">
```

Comment:
while this isn't exactly an issue (there are a couple sources that use this format), i do feel like we should be more consistent with these IDs.
```suggestion
	<element name="Descent into the Lost Caverns of Tsojcanth" type="Source" source="Core" id="ID_WOTC_SOURCE_DESCENT_INTO_THE_LOST_CAVERNS_OF_TSOJCANTH">
```